network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/TerminalNodeBreakerViewImpl.java

```diff
@@ -70,8 +70,13 @@ public void moveConnectable(int node, String voltageLevelId) {
             throw new ValidationException(attributes.getResource(), "an equipment (" + terminal.getConnectable().getId()
                     + ") is already connected to node " + node + " of voltage level " + voltageLevelId);
         }
-        attributes.setNode(node);
-        attributes.setVoltageLevelId(voltageLevelId);
+        getAbstractIdentifiable().updateResource(res -> {
+            InjectionAttributes attr = attributesGetter.apply(res);
+            attr.setConnectableBus(null);
+            attr.setBus(null);
```

Comment:
the same remarks for Bus and connectableBus

Reply:
and same answer as well.